other/upgrade_2-1_postgresql.sql

```diff
@@ -2681,6 +2694,12 @@ if (!array_key_exists($modSettings['smiley_sets_default'], $filtered))
 ---}
 ---#
 
+---# Drop obsolete filename column from smileys table
+ALTER TABLE {$db_prefix}smileys
+DROP COLUMN filename;
```

Comment:
i would like to see here ``DROP COLUMN IF EXISTS filename;``

Reply:
Good idea.